spec.md.html

```diff
@@ -550,6 +550,103 @@
   - Attenuation Distance
   - Subsurface Color
 
+Material Subtypes
+=================
+
+The Enterprise PBR Shading Model offers a limited but still substantial amount of parameters. For easier usability, especially for novice users, we recommend to use the following subtypes, which expose only subsets of parameters:
+* Car Paint
+* Emissive
+* Glass
+* Metal
+* Plastic
+* Textile/Leather
+* Wood
+
+For compatibility with the widely spread Unreal model [#Kar13], additionally a general purpose material called "Basic" is recommended.
+
+Tables [subtypes-parameters-metals], [subtypes-parameters-opaque-dielectrics] and [subtypes-parameters-transparent-dielectrics] describe recommended fixed and default values for these eight different material types. While fixed values should not be exposed to the user, we recommend to initialize all other values with the default values stated below. Color values are specified in linear sRGB space.
```

Comment:
```suggestion
Table [subtypes-parameters-metals], Table [subtypes-parameters-opaque-dielectrics] and Table [subtypes-parameters-transparent-dielectrics] describe recommended fixed and default values for these eight different material types. While fixed values should not be exposed to the user, we recommend to initialize all other values with the default values stated below. Color values are specified in linear sRGB space.
```
Links won't work without "Table" keyword